<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkItemEditor</class>
 <widget class="QDialog" name="BookmarkItemEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Bookmark</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Channel:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="channelsCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Day:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="dayOfWeekCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Any day</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Monday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tuesday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wednesday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thursday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Friday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Saturday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sunday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Monday to Friday</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Saturday and Sunday</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="otherDay">
         <property name="text">
          <string>Other ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Between:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTimeEdit" name="startTimeEdit">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="time">
          <time>
           <hour>18</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>and</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="stopTimeEdit">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="time">
          <time>
           <hour>23</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="anyTimeCheck">
         <property name="text">
          <string>Any time</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="colorSelect">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="seasonEnable">
       <property name="text">
        <string>Season:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="seasonEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>e.g., 1, 3-5, 7+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>channelsCombo</tabstop>
  <tabstop>dayOfWeekCombo</tabstop>
  <tabstop>otherDay</tabstop>
  <tabstop>startTimeEdit</tabstop>
  <tabstop>stopTimeEdit</tabstop>
  <tabstop>anyTimeCheck</tabstop>
  <tabstop>colorSelect</tabstop>
  <tabstop>seasonEnable</tabstop>
  <tabstop>seasonEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
